domain-xample/svc-hoppy-usage/src/async_client/main.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+from config import exchange_properties, request_queue_properties, reply_queue_properties, REQUEST_QUEUE, REPLY_QUEUE
+from hoppy.async_hoppy_client import AsyncPublisher, RetryableAsyncHoppyClient
+from hoppy.config import RABBITMQ_CONFIG
+from downstream_service import DownStreamService
+
+async_requester = AsyncPublisher(RABBITMQ_CONFIG, exchange_properties, request_queue_properties, REQUEST_QUEUE)
+async_client = RetryableAsyncHoppyClient(
+    app_id="xample",
+    name="xample-async-client",
+    config=RABBITMQ_CONFIG,
+    exchange_properties=exchange_properties,
+    request_queue_properties=request_queue_properties,
+    reply_queue_properties=reply_queue_properties,
+    request_routing_key=REQUEST_QUEUE,
+    reply_routing_key=REPLY_QUEUE,
+    max_latency=3,
```

Comment:
just to double-check: the desired `max_latency` is `3`?  The only reason I ask is `30` is the default 
https://github.com/department-of-veterans-affairs/abd-vro/blob/83c4aadb2922307c366fe8c0cd63d8e79027aa3e/shared/lib-hoppy/hoppy/async_hoppy_client.py#L196 .

Reply:
Not a typo! Just providing an example config that isn't the default!